configuration/appframework/referenceapplication/visitActions_extension.json

```diff
@@ -52,13 +52,32 @@
         "extensionPointId": "patientDashboard.visitActions",
         "type": "link",
         "label": "referenceapplication.app.capturevitals.title",
-        "url": "htmlformentryui/htmlform/enterHtmlFormWithSimpleUi.page?patientId={{patient.uuid}}&visitId={{visit.uuid}}&definitionUiResource=referenceapplication:htmlforms/vitals.xml",
+        "url": "htmlformentryui/htmlform/enterHtmlFormWithSimpleUi.page?patientId={{patient.uuid}}&visitId={{visit.uuid}}&definitionUiResource=file%3Aconfiguration%2Fhtmlforms%2Fvitals.xml",
```

Comment:
I don't think the `%3A` and `%2F` syntax is needed, and it's harder to read.  Let's use `:` and `/` unless proven that we can't.

Reply:
I think : and / refused to work but We really don't need this change to be used, I just did that for me to make sure that we can access html form from Overall actions or Active actions.

I will try to use : and / to see if it really can't work.

Thank you so much again, Mike.